### ISSUE TITLE
Huge increase of usb error codes [Golem 0.4.0] [MacOS 10.12.4]

### ISSUE THREAD

**Issue by apolikamixitos** [golemfactory/clay#971]
```
I0407 00:49:44.942492 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:49:54.942804 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:49:57.939926 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:50:10.942422 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:50:23.943716 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:50:24.941162 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:50:36.974703 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:50:49.942457 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:50:54.940353 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:51:02.940006 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:51:16.062512 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:51:24.942395 usb.go:93] handle_events: error: libusb: interrupted [code -10]
I0407 00:51:28.939552 usb.go:93] handle_events: error: libusb: interrupted [code -10]
```

**Comment by badb** [comment]
Bug connecte with geth synchronization.

**Comment by chfast** [comment]
This is caused by geth searching for hardware wallets.

**Comment by chfast** [comment]
geth 1.6.1 is released, so we can upgrade and add `--nousb` option.

**Comment by badb** [comment]
Resolved by #1071
